docs/migrating-from-mdl.md

```diff
@@ -0,0 +1,318 @@
+# Migrating from Material Design Lite
+
+Material Components for the web (MDC-Web) is the successor to the Material Design Lite (MDL) project.
+
+While the philosophy behind the two projects is quite similar, migrating to MDC-Web requires a number of changes, from
+class names to different DOM structures. In addition, there are several choices to be made regarding component
+initialization, how to depend on MDC-Web, and theming/styling mechanisms.
+
+This document attempts to summarize and guide you through the work involved. Let’s get started!
+
+> **Note:** If you’re thinking of migrating your application to MDC-Web, please bear in mind that it’s still in an alpha
+state and thus not recommended for production.
```

Comment:
I don't know if we want to say `not recommended for production`. We may want to say something like `APIs and certain UX features are subject to change`.